Modules/_datetimemodule.c

```diff
@@ -6830,7 +6897,7 @@ _datetime_exec(PyObject *module)
         return -1;
     }
 
-    PyDateTime_CAPI *capi = get_datetime_capi();
+    PyDateTime_CAPI *capi = get_datetime_capi(st);
```

Comment:
This function name is misleading. Can you please rename it to create_datetime_capi()?

Reply:
I agree, but I think we should do that in a separate PR. The `get_datetime_capi` is not introduced by this PR; it is already in place.